### ISSUE TITLE
Debian/Ubuntu: Build stops with "error: missing binary operator before token"

### ISSUE THREAD

**Issue by SantosSi** [manisandro/gImageReader#339]
Current Git build stops on Debian Testing and Ubuntu 18.04, but cross compilation for Windows in Docker Fedora image runs through:

```
.../gImageReader/qt/src/TessdataManager.cc:94:38: error: missing binary operator before token "("
 #if TESSERACT_VERSION >= MAKE_VERSION(4, 0, 0)
                                      ^
.../gImageReader/qt/src/TessdataManager.cc:134:38: error: missing binary operator before token "("
 #if TESSERACT_VERSION >= MAKE_VERSION(4, 0, 0)
                                      ^
CMakeFiles/gimagereader.dir/build.make:390: die Regel für Ziel „CMakeFiles/gimagereader.dir/qt/src/TessdataManager.cc.o“ scheiterte
```

**Comment by manisandro** [comment]
Probably something changed in tesseract, I'll need to test with latest tesseract when I have a moment.

**Comment by manisandro** [comment]
Fixed in 8e94c7de11bf77d14ae03f473abcf305f0d70772
